### PR TITLE
QuotesBear.py: change CAN_DETECT to CAN_FIX

### PR DESCRIPTION
**Pull request by Jakod96**
Changes CAN_DETECT to CAN_FIX in QuotesBear.py

Closes https://github.com/coala/coala-bears/issues/1571

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

---

### CODE REVIEW

File: =0.11.0.dev20170322155718 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Collecting coala
+  Downloading coala-0.10.0.tar.gz (2.0MB)
+    Complete output from command python setup.py egg_info:
+    coala supports only python 3.4 or later.
+
```
Comment (vijeth-aradhya) [inline =0.11.0.dev20170322155718]: Is this change related to the commit?

Comment (Jakod96) [inline =0.11.0.dev20170322155718]: Hi, it got added when I ran tests I belive, not sure. I will reread the documentation and attempt to update the pull request so it passes the tests and making sure it doesn't add unneeded files.

Comment (Jakod96) [inline =0.11.0.dev20170322155718]: Removed the file from the commit.

Comment (supergr35) [inline =0.11.0.dev20170322155718]: Not quite the way that you want to do it, don't use github UI for anything. Just change it locally and remove your other commit. Check the docs.

**Comment by supergr35** [comment]
Don't just change the title. Change the actual commit message. See docs.

**Comment by naman1901** [comment]
Hey. I had the same problem as you are having. Just do the following:-
1. `git commit --amend`
2. A text editor will open. Replace `Changed` to `Change`. Save and close.
3. 'git push origin <branchname> --force` 
This isn't the recommended way, but since you're making changes on your own branch, this will work just fine.

**Comment by jayvdb** [comment]
abandoned
